_people/amelia_gann.md

```diff
@@ -2,6 +2,12 @@
 title: Amelia Gann
 headshot: trLKLFk
 gender: female
-submitted: false
+submitted: 2018-01-18
+contact_allowed: true
 graduated: 2015
+course:
+ - English
 ---
+
+Currently studying at Academy of Live and Recorded Arts, London
+MA Acting (postgrad) Sept 2017 - December 2018
```

Comment:
Line break before to make new paragraph.

Reply:
Ritual comment already there, is down as award or tour on the show file

Reply:
Ah, ignore me then 😄